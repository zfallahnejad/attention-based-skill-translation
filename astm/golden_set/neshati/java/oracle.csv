10397,oracle
266304,oracle
330315,oracle
571407,oracle
70604,oracle
157882,oracle
119634,oracle
6568,oracle
21234,oracle
48503,oracle
103154,oracle
3093319,oracle
