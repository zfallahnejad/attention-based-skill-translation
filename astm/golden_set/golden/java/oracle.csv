10397,48,33,0.5925925925925926
266304,33,15,0.6875
330315,31,37,0.45588235294117646
571407,26,23,0.5306122448979592
70604,19,24,0.4418604651162791
157882,17,14,0.5483870967741935
6568,13,10,0.5652173913043478
21234,12,12,0.5
48503,11,5,0.6875
3093319,11,10,0.5238095238095238
103154,11,4,0.7333333333333333
1707091,10,7,0.5882352941176471
